### PR TITLE
Add deputy role to adherent admin

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: src/Admin/DistrictAdmin.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    {
+        $formMapper
+            ->add('name', TextType::class, [
+                'label' => 'Nom',
+                'filter_emojis' => true,
+            ])
+            ->add('countries', TextType::class, [
+                'label' => 'Pays',
+            ])
+            ->add('geoShape', TextType::class, [
```
Comment (ottaviano) [inline src/Admin/DistrictAdmin.php]: it's really needed this admin? if yes, do you try create one district?
I think a `TextGeoShapeDataTransformer` is missing here, no?

File: src/Admin/DistrictAdmin.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+class DistrictAdmin extends AbstractAdmin
+{
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
+            ->add('name', TextType::class, [
+                'label' => 'Nom',
+                'filter_emojis' => true,
+            ])
+            ->add('countries', TextType::class, [
```
Comment (ottaviano) [inline src/Admin/DistrictAdmin.php]: `CountryType` ?

File: src/Entity/Adherent.php [hunk 3]
```diff
@@ -1120,7 +1120,25 @@ public function getManagedDistrict(): ?District
 
     public function setManagedDistrict(?District $district): void
     {
+        if ($district
+            && $district->getAdherent()
+            && $district->getAdherent()->getUuid()->toString() !== $this->getUuid()->toString()) {
```
Comment (ottaviano) [inline src/Entity/Adherent.php]: `!$this->equals($district->getAdherent())`

File: src/Admin/DistrictAdmin.php (hunk 1 above)
Comment (illusionOfParadise) [reply src/Admin/DistrictAdmin.php]: No, it's needed only to modify a district from AdherentAdmin.

Comment (illusionOfParadise) [reply src/Admin/DistrictAdmin.php]: This admin is not shown in the admin dashboard and menu.

Comment (ottaviano) [reply src/Admin/DistrictAdmin.php]: oh I see, then the fields `countries` and `geoShape` are useless here, no? as you use only name field in AdherentAdmin.

**Pull request merged**